packages/network-of-terms-catalog/catalog/datasets/genre_filmmuseum.jsonld

```diff
@@ -0,0 +1,37 @@
+{

```

Comment:
Please use dashes `-` the filenames, as we already had for the old version of the GTAA endpoint: `genre-filmmuseum.jsonld`.

Reply:
Is it ok that I remove the older GTAA files and replace them with the newer ones, including dashes?

Reply:
Yes!